features/org.eclipse.e4.rcp/feature.xml

```diff
@@ -452,6 +462,17 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
```

Comment:
We can not include native win32 aarch fragments in features unless we have a builder in our JIPPs allowing to rebuild them on e.g. source change in SWT. Until this happens we can only have sources that can't end up in deliverables.

Reply:
OK then. I'll withdraw this PR and will consider it again when the build infrastructure for Windows Arm64 is in place.

Reply:
This part can't go in.